capi.config.ts

```diff
@@ -74,5 +74,8 @@ export const config: CapiConfig = {
         },
       },
     },
+    polkadotFromMetadata: {
+      metadata: await Deno.readFile("examples/raw_rpc/metadata"),
+    },
```

Comment:
I suppose it would demonstrate something along the lines of...

```ts
import { PolkadotFromMetadataChainRune } from "@capi/polkadot-from-metadata"

const chain = PolkadotFromMetadataChainRune.from(WsConnection.bind("ws://..."))
```

... I'm not sure if it would be a useful example. Thoughts?

Reply:
I find that it would be helpful to demonstrate how to use a custom endpoint.
And it will add an extra check in CI